spec/find-view-spec.coffee

```diff
@@ -844,6 +844,37 @@ describe 'FindView', ->
           atom.commands.dispatch(findView.findEditor.element, 'core:confirm')
           expect(findView.descriptionLabel).toHaveClass 'text-error'
 
+      describe "when there are existing selections", ->
+        describe "when any existing selections match the pattern", ->
+          it "shouldn't jump to the next match when toggled on", ->
```

Comment:
Usually we avoid the `should` phrasing in the specs, opting for the slightly more direct `it "does not jump to the next match when toggled on"`.